### ISSUE TITLE
Help, I can't read the SQL error

### ISSUE THREAD

**Issue by nssatlantis** [Ombi-app/Ombi#4830]
**Describe the bug**
Unable to read the SQL Error on the cmd screen.

**To Reproduce**
Open Ombi.exe after update
notice all SQL things are fucked and it spits an error.
Try to read the error but its closing faster than the speed of light so good luck with that.
Get angry because github bot forces you to use this format.

**Expected behavior**
I need/want to be able to read that goddamn error message so that I can re-do all the ombi settings because I had to delete all the databases because it refused to work.

**Screenshots**
I could never screenshot that error in time, so yeah.

**Logs (Logs directory where Ombi is located)**
If it would show up in the logfile, I wouldn't really be here now would I?

**Desktop (please complete the following information):**
 - OS: Windows.

**Ombi Version (please complete the following information):**
 - Version Your absolute latest and finest..
- Media Server No idea
- Database Type: MariaDB MySQL, but without a database at this point so it doesn't matter

**Additional context**
Sorry, I am just reasonably frustrated to the point of pissed, how one update can break the entire Ombi setup and force me to spend time in my weekend to fix this..... And I can't even read the goddamn sql error message.

**Comment by nssatlantis** [comment]
![image](https://user-images.githubusercontent.com/11834002/208246267-a5150a88-f30b-4d31-8fa7-d0d6a1f220ef.png)

I have managed to get a screenshot in of the error, not sure where to go from here as it worked perfectly fine before..

**Comment by tidusjar** [comment]
Did you close this on purpose?

first of all, why did you delete your database? Makes no sense. Anyway,
What very were you running and what version did you update to?

what’s your MySQL version?

**Comment by nssatlantis** [comment]
> Did you close this on purpose?
> 
> first of all, why did you delete your database? Makes no sense. Anyway,
> What very were you running and what version did you update to?
> 
> what’s your MySQL version? 

End of the day shit seemes to have broken, and re-creating was simpler..
I also made daily backups, but due to a config issue it hadn't been making these backups for months now and it had deleted the older ones.

Anyhow, the issue was that I was on an older version of mariadb, which I only found after digging through your previous issues. So either I was just blind at the time or it could be placed somewhere else so that it is more visible.

But yeah, issue is fixed, ombi still works like magic.
